docs/components/utilities/cryptography.md

```diff
@@ -0,0 +1,332 @@
+# Cryptography
+
+## Overview
+
+The **Cryptography** component includes a selection of algorithms for hashing, encoding, and decoding data.
+
+## Properties
+
+### AES-128 Key
+
+\[ Getter + Setter \]  
+`Type: Text`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'AES-128 Key')]]
+
+The key used for encoding and decoding with AES-128. Ideally a hexadecimal string of length 16.
+
+### AES-256 Key
+
+\[ Getter + Setter \]  
+`Type: Text`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'AES-256 Key')]]
+
+The key used for encoding and decoding with AES-256. Ideally a hexadecimal string of length 32.
+
+### BCrypt Salt Size
+
+\[ Getter + Setter \]  
+`Type: Number`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'BCrypt Salt Size')]]
+
+The size of the salt used by BCrypt. Has a value of 10 by default.
+
+### PBKDF2 Hash Byte Size
+
+\[ Getter + Setter \]  
+`Type: Number`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'PBKDF2 Hash Byte Size')]]
+
+The size of the PBKDF2 Hash in Bytes. Has a value of 18 by default.
+
+### PBKDF2 Iterations Number
+
+\[ Getter + Setter \]  
+`Type: Number`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'PBKDF2 Iterations Number')]]
+
+The number of iterations of PBKDF2 to run. Has a value of 64000 by default.
+
+### PBKDF2 Salt Byte Size
+
+\[ Getter + Setter \]  
+`Type: Number`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'PBKDF2 Salt Byte Size')]]
+
+The size of the salt used by PBKDF2 in Bytes. Has a value of 24 by default.
+
+### TripleDES Key
+
+\[ Getter + Setter \]  
+`Type: Text`
+
+[[PropertyBlockGetterAndSetter('Cryptography', 'TripleDES Key')]]
+
+The key used for encoding and decoding with TripleDES. The key must be a string of 48 hexadecimal characters.
+
+## Methods
+
+### AES-128 Decode
+
+\[ Text `AES-128 Hash` \]  
+
+[[Method('Cryptography', 'AES-128 Decode', 'AES-128 Hash')]]
```

Comment:
The syntax for the method macro is: `[[Method('Component Name', 'Method Name', <has_return_value: Boolean>, params)]]`

This should be `[[Method('Cryptography', 'AES-128 Decode', True, 'AES-128 Hash')]]` , so that the rendered method block is similar to the one on the Creator with the return value.
```suggestion
[[Method('Cryptography', 'AES-128 Decode', True, 'AES-128 Hash')]]
```